src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -3,184 +3,245 @@
+import { WORKSPACE_CREATE_APP_ID, WORKSPACE_LIST_APP_ID } from '../../../common/constants';
 import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { WorkspaceUseCase } from '../../types';
+import { WorkspaceUseCaseCard } from './workspace_use_case_card';
+import { WorkspaceUseCaseFlyout } from '../workspace_form';
+import { navigateToWorkspacePageWithUseCase } from '../utils/workspace';
 
-export const WorkspaceInitial = () => {
+export interface WorkspaceInitialProps {
+  registeredUseCases$: BehaviorSubject<WorkspaceUseCase[]>;
+}
+
+export const WorkspaceInitial = ({ registeredUseCases$ }: WorkspaceInitialProps) => {
   const {
-    services: { application, chrome, uiSettings },
+    services: { application, chrome, workspaces, http },
   } = useOpenSearchDashboards<CoreStart>();
-  const isDashboardAdmin = application.capabilities.dashboards?.isDashboardAdmin;
-  const logos = chrome.logos;
-  const createWorkspaceUrl = application.getUrlForApp(WORKSPACE_CREATE_APP_ID, { absolute: true });
-  const settingsAndSetupUrl = application.getUrlForApp('settings_landing', { absolute: true });
-  const isDarkTheme = uiSettings.get('theme:darkMode');
-  const backGroundUrl = isDarkTheme ? BackgroundDarkSVG : BackgroundLightSVG;
+  const isDashboardAdmin = !!application.capabilities.dashboards?.isDashboardAdmin;
+  const availableUseCases = registeredUseCases$
+    .getValue()
+    .filter((item) => !item.systematic || item.id === 'all');
+  const workspaceList = workspaces.workspaceList$.getValue();
+  const [isUseCaseFlyoutVisible, setIsUseCaseFlyoutVisible] = useState(false);
+  const [defaultUseCaseId, setDefaultUseCaseId] = useState(availableUseCases[0].id);
+
+  const handleClickUseCaseInformation = useCallback((useCaseId: string) => {
+    setIsUseCaseFlyoutVisible(true);
+    setDefaultUseCaseId(useCaseId);
+  }, []);
+  const handleFlyoutClose = useCallback(() => {
+    setIsUseCaseFlyoutVisible(false);
+  }, []);
+
+  const useCaseCards = availableUseCases.map((useCase) => {
+    return (
+      <EuiFlexItem key={useCase.id}>
+        <WorkspaceUseCaseCard
+          useCase={useCase}
+          workspaces={workspaceList}
+          application={application}
+          http={http}
+          isDashboardAdmin={isDashboardAdmin}
+          handleClickUseCaseInformation={handleClickUseCaseInformation}
+        />
+      </EuiFlexItem>
+    );
+  });
+
+  const mountUserAccountRef = useRef<HTMLDivElement>(null);
+  const mountSettingRef = useRef<HTMLDivElement>(null);
+  const mountDevToolsRef = useRef<HTMLDivElement>(null);
+
+  const [isCreateWorkspacePopoverOpen, setIsCreateWorkspacePopoverOpen] = useState(false);
+  const [userAccountMount, setUserAccountMount] = useState<ChromeNavControl | undefined>(undefined);
+  const [settingMount, setSettingMount] = useState<ChromeNavControl | undefined>(undefined);
+  const [devToolsMount, setDevToolsMount] = useState<ChromeNavControl | undefined>(undefined);
```

Comment:
What about we just use `const items = useObservable(chrome.navControls.getLeftBottom$());` and cherry-pick the needed icon to render? the useState seems overwhelming here.

Reply:
Using `useObservable` results in the error `Maximum update depth exceeded`.
![Uploading image.png…]()

Reply:
Perhaps we can
```
const leftBottom$Ref = useRef(chrome.navControls.getLeftBottom$());
const items = useObservable(leftBottom$Ref.current);
```
I guess it is because every time we cal getLeftBottom it will return a new observable.